pxc/jenkins/pxc-package-testing.groovy

```diff
@@ -40,16 +40,17 @@ void runMoleculeAction(String action, String version, String scenario) {
         sh """
             source venv/bin/activate
             export MOLECULE_DEBUG=0
-            export install_repo=${params.INSTALL_REPO}
+            export install_repo=${params.install_repo}
+            export pxc57repo=${params.pxc57_repo}
```

Comment:
We need this var only for pxc57 tests so on old jenkins we export it only for pxc57, please do here the same.

Reply:
Hi,

Added a conditional block for setting pxc57repo variable.